packages/browser/__tests__/login/oidc/redirectHandler/AuthCodeRedirectHandler.spec.ts

```diff
@@ -170,11 +270,64 @@ describe("AuthCodeRedirectHandler", () => {
       await redirectInfo.fetch("https://some.other.url");
       // @ts-ignore
       const header = fetch.fetch.mock.calls[0][1].headers["Authorization"];
-      // We test that the Authorization header matches the structure of a JWT.
       expect(
         // @ts-ignore
         header
-      ).toMatch(/^Bearer .+\..+\..+$/);
+      ).toMatch(/^Bearer some token$/);
```

Comment:
I was a bit confused by this test. I was assuming it would do an unauthenticated fetch by default, but apparently some token is being passed in here (`some token`), but it's not clear to me where it comes from. Presumably from one of the mocks, but if it's not too much effort, maybe that value ("some token") can be explicitly passed to the mocks from this test, so it's clear where the assertion came from?

Reply:
Since the value is returned by the mocked module, it's a bit hard to explicitly pass the value as part of the test, but I added a comment to clarify things a bit in ba04ef918e13a08a019f8d5cb1556c808cb8a415